bootcamp-milestone-2/src/app/blog/page.tsx

```diff
@@ -0,0 +1,40 @@
+import styles from './page.module.css'
+import "@/app/global.css"
+import blogs from "@/app/blogData";
+import BlogPreview from "@/app/components/blogPreview";
+
+
+export default function Resume() {
+  return (
+    <div>
+  <main>
+
+    <main className="main">
+        <div className="content" id="blog">
+            <ul id="blog-list" className="blog-list"></ul>
+        </div>
+    </main>
+    <h1 className="page-title">MY BLOG</h1>
+    Hello Guys! This is my blog.
+  </main>
+
+  {blogs.map((blog) => (
```

Comment:
You are missing individual blog pages. Marking complete for now since m3a will require it.